en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
-4. **Dynamic groups based on free-form search expressions** contain entries that match a specified search expression, using the same syntax as the [search panel](search.md) on the side pane. This [syntax](search.md#advanced) supports logical operators \(`AND`, `OR`, `NOT`\) and allows to specify one or more fields to search, facilitating more flexible group definitions than a keyword search \(e.g. `author=smith and title=electrical`\).
+To test this, create two groups having some entries in common. Click the **Intersection/Union** button and make sure that **Union** is selected.
+Now select both groups.
+You should see all entries contained in any of the two groups. Click again on the **Intersection/Union** button and select **Intersection**.
+Now you should see only those entries contained in both groups (which might be none at all if groups do not share entries, or exactly the same entries if both groups contain the same entries).
 
-Every group that you create is of one of the last three types. The group editing dialog, which is invoked by double-clicking on a group, shows a short description of the selected/edited group in plain English.
+## Group tree. Creating and removing groups
 
-## Groups structure, creating and removing groups
+Just like directories, groups are structured like a tree, with the group *All Entries* at the root.
+By right-clicking on a group and selecting **Add subgroup**, you can add a subgroup to the selected group.
+The **+** button (near the top of the pane) lets you create a new subgroup of the group **All Entries**, regardless of the currently selected group(s).
+The context menu also allows removing groups and/or subgroups, and to sort subgroups alphabetically.
+Moving groups to a different location in the tree can be done by drag-and-drop.
 
-Just like directories, groups are structured like a tree, with the group **All Entries** at the root. By right-clicking on a group you can add a new group to the tree, either at the same level as the selected group or as a subgroup of it. The **New Group** button lets you create a new subgroup of the group **All Entries**, regardless of the currently selected group\(s\). The context menu also allows to remove groups and/or subgroups, to sort subgroups alphabetically, or to move groups to a different location in the tree. The latter can also be done by Drag and Drop, with the restriction that Drag and Drop does not support changing the order of a group's subgroups.
+Undo and redo is supported for all edits. **%%%Help needed: Not true in 5.2. Should this be considered as a bug or a feature?%%%**
```

Comment:
It's a known bug. Maybe remove the sentence about undo/redo completely since it's not that important?

Reply:
Sentence removed.